### PR TITLE
Add missing newline for encoded version variables.

### PR DESCRIPTION
**Pull request by abrisco**
Currently lockfiles are encoded as the following due to a missing newline in https://github.com/rustsec/rustsec/pull/596:
```toml
# This file is automatically @generated by Cargo.
# It is not intended for manual editing.
version = 3
[[package]]
```

This change fixes the encoding to correctly match Cargo generated `Cargo.lock` files
```toml
# This file is automatically @generated by Cargo.
# It is not intended for manual editing.
version = 3

[[package]]
```

### CODE REVIEW

**Comment by tarcieri** [comment]
Thanks! Perhaps it would be good to add a test that it round-trips a lockfile with a `version` back to the original encoding?

File: cargo-lock/tests/examples/Cargo.lock.v1 [hunk 1]
```diff
@@ -1,6 +1,5 @@
-# Example non-trivial `Cargo.lock` file.
```
Comment (abrisco) [inline cargo-lock/tests/examples/Cargo.lock.v1:1]: I'm not sure about the V1 lockfiles here. Did they normally get generated with this unique header?

**Review by abrisco — COMMENTED** [review]
> Thanks! Perhaps it would be good to add a test that it round-trips a lockfile with a version back to the original encoding?

How's this?

**Review by tarcieri — APPROVED** [review]

File: cargo-lock/tests/examples/Cargo.lock.v1 (hunk 1 above)
Comment (abrisco) [reply cargo-lock/tests/examples/Cargo.lock.v1:1]: Thanks! that's exactly what I did, just wasn't sure if it was the correct thing to do 😄

**Pull request merged**